manifests/supervisorcluster/1.20/cns-csi.yaml

```diff
@@ -104,6 +104,34 @@ subjects:
     kind: Group
     name: system:serviceaccounts:vmware-system-csi
 ---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vsphere-admin-csi-role
+rules:
+  - apiGroups: ["cns.vmware.com"]
+    resources: ["cnsregistervolumes"]
+    verbs: ["get", "list", "create", "delete", "watch"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "update", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["update", "get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: wcp:administrators:cluster-edit-csirole
+subjects:
+- kind: Group
+  name: sso:Administrators@<sso_domain>
```

Comment:
Is `<sso_domain>` populated by the WCP service deploying our driver?

Reply:
Yes, that change will be in WCP service. I will add it when we cherry-pick this to WCP.

Reply:
Yes, that change will be in WCP service. I will add it when we cherry-pick this to WCP.